JS/closures.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>A function  displayName() can access variables defined inside the function pam() :</p>
+
+<!-- <button type="button" onclick="pam()">Click Me!</button> -->
+<!-- <p id="demo2"></p> -->
+<p id="demo"></p>
+
+
+<script>
```

Comment:
Always create a new file.
Move the below code to a new file and link it on this page.

Reply:
Sure i will